2_drivers.yml

```diff
@@ -17,13 +17,39 @@ targets:
   #out_data/placeholder_for_gefs_archive:
    # command: copy_files()
     
-  #out_data/palceholder_for_nldas:
-   # command: copy_files()
-   
+  # NLDAS Files
+  ## NLDAS_time[0.377900]_x[397]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[397]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[397]_y[136].csv',
+    to = target_name)
+  ## NLDAS_time[0.377900]_x[401]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[401]_y[136].csv:
+    command: file.copy(from = 'in_data/NLDAS_time[0.377900]_x[401]_y[136].csv',
+    to = target_name)
+  
+  ## NLDAS to reservoir meteo rds  
+  log/reservoir_meteo_files.yml:
+    command: fetch_meteo_files(
+      out_yml = target_name,
+      nml_rds = 'out_data/reservoir_nml_list.rds')
+  
+  # GEFS archive
+  out_data/met_drivers.zip:
+    command: utils::zip(files = 'in_data/met_drivers', zipfile = target_name)
```

Comment:
this should the way to zip folders in the repo. Have we used any other function build anything functions in `/src/` for zipping existing folders?